_data/tools.yml

```diff
@@ -747,6 +747,15 @@
   description: Generate OAS files from code comments and easily host them ($ npm install oas -g)
   v2: true
   v3: true
+  
+- name: Generate OAS
+  category: miscellaneous
+  language: JavaScript, Java, Python, Ruby, Go, Typescript
```

Comment:
the project looks like a SaaS or JS platform?

Reply:
@acunniffe can you fix this?

Reply:
@matthewtrask taken care of, I didn't realize that 'SaaS' was the convention. Thanks